### ISSUE TITLE
MPD Client Spotify search

### ISSUE THREAD

**Issue by fr43nk** [mopidy/mopidy#1027]
I'm using the MPDroid client. I want to do a search including all the services that are activated in my mopidy server. But it looks like the results are only from the local music database. There is nothing included from i.e. spotify.
I already installed mopidy-spotify-tunigo but that's only a kind of playlist search (which doesn't get searched either)

Is there something I have to configure to get the search working?

Best regards

**Comment by trygveaa** [comment]
Have you installed `mopidy-spotify` and provided your credentials in the config?

**Comment by fr43nk** [comment]
Yes I have provided them. Spotify is working I can browse the playlists but I cannot search for a specific artist or title in spotify. It only searches my local music library.

**Comment by jodal** [comment]
Does the search work as expected when you use another client?

**Comment by joemarshall** [comment]
I think this may be a side effect of the way that mpdroid uses the list artist <blah> commands to list the artists, which then runs into the fact that the list commands are implemented via a full library scan, which obviously isn't implemented for spotify?
